src/components/PluginCommandsSection.vue

```diff
@@ -19,7 +21,7 @@
       <p>{{ command.description }}</p>
       <pre
         class="prose language-bash rounded-md"
-      ><code>meltano invoke {{ name }}:{{ command.name }} [args...]</code></pre>
+      ><code>meltano invoke {{ name }}:{{ command.name }}</code></pre>
```

Comment:
This is correct as written, and I think noting `args...` as an optional input is helpful so users can discover this feature.

Should we revert this part back?

```suggestion
      ><code>meltano invoke {{ name }}:{{ command.name }} [args...]</code></pre>
```

As noted in the PR thread, the added args are freely supported in `invoke` - just not in `run`.